packages/fxa-content-server/app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -48,7 +48,7 @@ export default {
   },
 
   isPanelOpen() {
-    return this.model.get('isPanelOpen');
+    return !!this.model.get('isPanelOpen');
```

Comment:
This one is from b7ed82f8534a7cc26755a09eb41dfd610f3cb1d0, `isPanelOpen` previously returned the boolean result of `hasClass`.

`app/tests/spec/views/settings/email.js` was expecting `false` but getting `undefined` instead.